sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -1275,7 +1275,7 @@ export class CertificateClient {
    */
   public async importCertificate(
     certificateName: string,
-    certificateValue: Uint8Array,
+    certificateBytes: Uint8Array,
```

Comment:
This change looks good! I was expecting to see a change in the API review file too. Is the plan to generate it later?